research/picai/preprocessing_transforms.py

```diff
@@ -33,7 +33,7 @@ def resample_img(
     original_size = image.GetSize()
 
     # convert PICAI z, y, x (Depth x Height x Width) convention to SimpleITK's convention x, y, z
-    out_spacing = list(reversed(spacing))
+    out_spacing = list(reversed(list(spacing)))
```

Comment:
I know reversed was my suggestion. However, now that we've moved from sequences to tuples, maybe we just do the following?
```
out_spacing = (spacing[2], spacing[1], spacing[0])
```